src/main/scala/org/opennetworkinsight/dns/DNSPreLDA.scala

```diff
@@ -32,24 +32,9 @@ object DNSPreLDA {
 
     import sqlContext.implicits._
     val feedbackFile = scoresFile
-    var time_cuts = new Array[Double](10)
-    var frame_length_cuts = new Array[Double](10)
-    var subdomain_length_cuts = new Array[Double](5)
-    var numperiods_cuts = new Array[Double](5)
-    var entropy_cuts = new Array[Double](5)
-    var df_cols = new Array[String](0)
-
-    val l_top_domains = Source.fromFile("top-1m.csv").getLines.map(line => {
-      val parts = line.split(",")
-      val l = parts.length
-      parts(1).split("[.]")(0)
-    }).toSet
-    val top_domains = sc.broadcast(l_top_domains)
-
-
     val scoredFileExists = new java.io.File(feedbackFile).exists
 
-    val falsePositives: org.apache.spark.rdd.RDD[String] = if (scoredFileExists) {
+    val falsePositives: DataFrame = if (scoredFileExists) {
 
       /* dns_scores.csv - feedback file structure
```

Comment:
maybe for another check-in.... but the feedback pieces of this code seems to generate so much code and complication, it might be best to hide as much of the feedback code in another function or even object if at all possible, to abstract away the details of how feedback is working